src/foundation.cs

```diff
@@ -8569,6 +8570,14 @@ public partial interface NSItemProvider : NSCopying {
 		[Export ("preferredPresentationSize")]
 		CGSize PreferredPresentationSize { get; }
 #endif
+
+		//[Mac (10,12)][Async]
+		//[Export ("registerCloudKitShareWithPreparationHandler:")]
+		//void RegisterCloudKitShare (Action<Action<CKShare, CKContainer, NSError>> preparationHandler);
+
+		//[Mac (10,12)]
+		//[Export ("registerCloudKitShare:container:")]
+		//void RegisterCloudKitShare (CKShare share, CKContainer container);
```

Comment:
easy to forget and hard to grep later, add a TODO, or keep it in the document of things to bind